dashboard/app/models/contact_rollups_raw.rb

```diff
@@ -23,12 +23,134 @@ def self.extract_parent_emails
     source_sql = <<~SQL
       SELECT parent_email, MAX(updated_at) AS updated_at
       FROM users
+      WHERE parent_email > ''
       GROUP BY parent_email
     SQL
     query = get_extraction_query(source_sql, 'parent_email', [], true, 'dashboard.users.parent_email')
     ContactRollupsV2.execute_query_in_transaction(query)
   end
 
+  def self.extract_user_geos
+    teacher_query = <<~SQL
+      SELECT id, email
+      FROM users
+      WHERE user_type = 'teacher' AND email > ''
+    SQL
+
+    # An user can have many user_geos records. user_geos records starts with only NULL
```

Comment:
No change needed, but just noting this almost never happens -- looks like we have a total of 2600 accounts with multiple records in `user_geos`.